latch_cli/main.py

```diff
@@ -437,11 +437,24 @@ def execute(
 ):
     """Drops the user into an interactive shell from within a task."""
 
-    from latch_cli.services.execute.main import exec
+    from latch_cli.services.k8s.execute import exec
 
     exec(execution_id=execution_id, egn_id=egn_id, container_index=container_index)
 
 
+@main.command("attach")
```

Comment:
maybe name should be more specific. or there is a group of command namespaced under `nextflow`. can add it there so that we have `latch nextflow attach --execution_id`